scripts/src/sanitycheckpr/sanitycheckpr.py

```diff
@@ -17,23 +23,69 @@ def ensure_only_chart_is_modified(api_url):
     files_api_url = f'{api_url}/files'
     headers = {'Accept': 'application/vnd.github.v3+json'}
     r = requests.get(files_api_url, headers=headers)
-    pattern = re.compile(r"charts/(\w+)/([\w-]+)/([\w-]+)/([\w\.]+)/.*")
+    pattern = re.compile(r"charts/(\w+)/([\w-]+)/([\w-]+)/(.+?(?=\/))/.*")
     reportpattern = re.compile(r"charts/(\w+)/([\w-]+)/([\w-]+)/([\w\.]+)/report.yaml")
     count = 0
+    match_found = False
     for f in r.json():
-        if not pattern.match(f["filename"]):
-            print("PR should only modify chart related files")
+        filename = f["filename"]
+        match = pattern.match(filename)
+        if not match:
+            msg = f"[ERROR] PR should only modify chart related files: {filename}"
+            print(msg)
+            print(f"::set-output name=sanity-error-message::{msg}")
             sys.exit(1)
-        elif reportpattern.match(f["filename"]):
+        elif match:
+            if not match_found:
+                pattern_match = match
+                match_found = True
```

Comment:
"elif match" should be "else" 
given match is True "pattern_match=True" is easier to follow.

Reply:
Done. bf4863c47c0f01ad2f4bfe80a388d432e6d621b9